### PR TITLE
Add an npm start script to project template

### LINKED ISSUE #203: Add npm start script to example templates
After creating a project with the CLI the instructions below are displayed to the user. Typing npm start in the example root causes an error because there is no npm start script. 

To run the example:
  cd sudoku
  npm run test
  npm run build
  npm run start

### PR DESCRIPTION
**Pull request by ymekuria**
**Description**
Fixes
#203 

Add npm start script to the project template for example projects and update example project comment instructions. Currently the user is shown these instructions after creating an example project. 

To run the example:
  cd sudoku
  npm run test
  npm run build
  npm run start

Now a user can type npm start to run an example.

**Tested**
Successfully used this version of the CLI to create new projects without a start script, and new example projects with a start script on the package.json.

### CODE REVIEW

File: src/lib/example.js [hunk 1]
```diff
@@ -135,15 +135,32 @@ async function setProjectName(projDir) {
...
 }
 
+/**
+ * Helper to add start script to package.json.
+ * @param {string} file    Path to file
+ * @param {string} script  Start script to add.
+ */
+function addStartScript(file) {
+  const startScript = 'node build/src/index.js';
+  let packageJsonContent = JSON.parse(fs.readFileSync(file, 'utf8'));
```
Comment (jasongitmail) [inline src/lib/example.js:157]: We're using [fs-extra](https://www.npmjs.com/package/fs-extra), so we can simplify the value to `fs.readJsonSync(file)` ([src](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/readJson-sync.md))

File: src/lib/example.js [hunk 2]
```diff
@@ -135,15 +135,32 @@ async function setProjectName(projDir) {
...
+
+  // Add start script to template package.json
+  packageJsonContent['scripts']['start'] = startScript;
+  fs.writeFileSync(file, JSON.stringify(packageJsonContent, null, 2));
```
Comment (jasongitmail) [inline src/lib/example.js:161]: [`fs.writeJsonSync()`](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/writeJson-sync.md)

File: src/lib/example.js [hunk 3]
```diff
@@ -135,15 +135,32 @@ async function setProjectName(projDir) {
...
   spin.succeed(_green(step));
 }
 
+/**
+ * Helper to add start script to package.json.
+ * @param {string} file    Path to file
+ * @param {string} script  Start script to add.
+ */
+function addStartScript(file) {
+  const startScript = 'node build/src/index.js';
```
Comment (jasongitmail) [inline src/lib/example.js:156]: Would probably simplify to not use a var. Should be `'node build/src/run.js'` now right?

**Review by jasongitmail — CHANGES_REQUESTED** [review]

File: src/lib/example.js [hunk 4]
```diff
@@ -135,15 +135,30 @@ async function setProjectName(projDir) {
...
 }
 
+/**
+ * Helper to add start script to package.json.
+ * @param {string} file Path to file
+ */
+function addStartScript(file) {
+  let packageJsonContent = fs.readJsonSync(file, 'utf8');
+
+  // Add start script to template package.json
```
Comment (jasongitmail) [inline src/lib/example.js:157]: small nit: seems redundant to have this given how small the method is

**Review by jasongitmail — APPROVED** [review]
LGTM!

**Comment by ymekuria** [comment]
A start script was added in [this](https://github.com/o1-labs/zkapp-cli/pull/204) PR.

**Pull request merged**